AppController/djinn.rb

```diff
@@ -4042,6 +4042,16 @@ def rsync_files(dest_node)
     HelperFunctions.shell("rsync #{options} #{lib}/* root@#{ip}:#{lib}")
     HelperFunctions.shell("rsync #{options} #{app_task_queue}/* root@#{ip}:#{app_task_queue}")
     HelperFunctions.shell("rsync #{options} #{scripts}/* root@#{ip}:#{scripts}")
+    if dest_node.is_appengine?
+      locations_json = "/root/.appscale/locations-#{@options['keyname']}.json"
```

Comment:
Please avoid hardcoded paths.
If /root and /root/.appscale are not already available as constants, feel free to add them.